src/libcmd/repl.cc

```diff
@@ -270,6 +270,7 @@ void NixRepl::mainLoop()
             // ctrl-D should exit the debugger.
             state->debugStop = false;
             state->debugQuit = true;
+            std::cout << std::endl;
```

Comment:
```suggestion
            logger->cout("");
```
in order to avoid potential problems with the progress bar.